docs/API/plain_rules.md

```diff
@@ -106,12 +106,18 @@ from iotEvent
 where (cast(cast(bloodPressure?,String),double)>1.5 and type="BloodMeter")]
 ```
 
+You should take into consideration the following guidelines:
+
 -   Include `*,` in EPL select clause
--   The event stream from which take events must be **iotEvent**
--   A _type=_ condition may be concatenated for avoiding mixing different kinds of entities
--   The variable 'ruleName' in automatically added to the action, even if it is not present in the EPL text. The
-    ruleName automatically added this way is retrieved as part of the EPL text when the rule is recovered using GET
-    /rules or GET /rules/{name}.
+-   The event stream from which take events must be `iotEvent`
+-   A `type=` condition may be concatenated for avoiding mixing different kinds of entities
+-   Entity's attributes must be cast to `float` in case of being numeric, to `String` otherwise.
+-   All the attributes of the context broker notification are available as dynamic properties of the event object `ev`.
+-   A question mark `?` is _necessary_ for EPL to refer to ‘dynamic’ values (duck typing), such as `ev.id?`.
```

Comment:
This 3 new points came from `docs/user/index.md` updated in #592 but simplified here and moved to *source file*